### ISSUE TITLE
%(format_id)s isn't carried over to the merged file when special names are used (e.g. bestvideo+bestaudio)

### ISSUE THREAD

**Issue by endumiuz** [rg3/youtube-dl#4824]
```
[debug] System config: []
[debug] User config: []
[debug] Command-line args: ['--verbose', '--format', 'bestvideo+bestaudio', '--output', '%(title)s-%(format_id)s.%(ext)s', 'http://youtu.be/VY7L2HiP_0U']
[debug] Encodings: locale cp1252, fs mbcs, out cp850, pref cp1252
[debug] youtube-dl version 2015.01.30.2
[debug] Python version 2.7.8 - Windows-7-6.1.7601-SP1
[debug] exe versions: ffmpeg N-69233-g1f13348
[debug] Proxy map: {}
[youtube] VY7L2HiP_0U: Downloading webpage
[youtube] VY7L2HiP_0U: Extracting video information
[youtube] VY7L2HiP_0U: Downloading DASH manifest
[debug] Invoking downloader on 'https://r6---sn-5hn7ym7l.googlevideo.com/videoplayback?id=558ecbd8788fff45&itag=136&source=youtube&requiressl=yes&mm=31&ms=au&pl=19&nh=EAE&mv=m&ratebypass=yes&mime=video/mp4&gir=yes&clen=9135639&lmt=1386471197698097&du
r=304.387&upn=OBsfckXm4JI&signature=6967674DDB663E158705E315463F777AB358FDDA.90D3C69A9C59E938D387B06514E7A3BB25456F68&key=dg_yt0&fexp=900718,904846,907263,913444,919148,927622,930676,931376,934952,934953,934954,9405645,943917,947225,948124,952302,952
605,952901,955301,957201,959701&mt=1422651613&sver=3&ip=194.117.188.224&ipbits=0&expire=1422673255&sparams=ip,ipbits,expire,id,itag,source,requiressl,mm,ms,pl,nh,mv,ratebypass,mime,gir,clen,lmt,dur'
[download] Destination: d.notive - Taking Flight-136.f136.mp4
[download] 100% of 8.71MiB in 00:03
[debug] Invoking downloader on 'https://r6---sn-5hn7ym7l.googlevideo.com/videoplayback?id=558ecbd8788fff45&itag=141&source=youtube&requiressl=yes&mm=31&ms=au&pl=19&nh=EAE&mv=m&ratebypass=yes&mime=audio/mp4&gir=yes&clen=9708221&lmt=1386471144347779&du
r=304.483&upn=OBsfckXm4JI&signature=94C79A3AB5A7D71DB2C50989D806446AFD488EF1.8822BECEA4F1B6BB3321AF2FB5BC7786DDD3BC07&key=dg_yt0&fexp=900718,904846,907263,913444,919148,927622,930676,931376,934952,934953,934954,9405645,943917,947225,948124,952302,952
605,952901,955301,957201,959701&mt=1422651613&sver=3&ip=194.117.188.224&ipbits=0&expire=1422673255&sparams=ip,ipbits,expire,id,itag,source,requiressl,mm,ms,pl,nh,mv,ratebypass,mime,gir,clen,lmt,dur'
[download] Destination: d.notive - Taking Flight-141.f141.m4a
[download] 100% of 9.26MiB in 00:21
[ffmpeg] Merging formats into "d.notive - Taking Flight-bestvideo+bestaudio.mp4"
[debug] ffmpeg command line: ffmpeg -y -i 'd.notive - Taking Flight-136.f136.mp4' -i 'd.notive - Taking Flight-141.f141.m4a' -c copy -map 0:v:0 -map 1:a:0 'd.notive - Taking Flight-bestvideo+bestaudio.mp4'
```
I was expecting the merged file to be named "d.notive - Taking Flight-136+141.mp4" not "d.notive - Taking Flight-bestvideo+bestaudio.mp4"

**Comment by dstftw** [comment]
Thanks for the report, it will be fixed in the next version.

**Comment by endumiuz** [comment]
Wow, you are fast. Thank you.

**Comment by phihag** [comment]
I believe this issue to be fixed in youtube-dl 2015.02.02. See [our FAQ](https://yt-dl.org/update) if you need help updating.
